packages/neon/neon_files/lib/models/file_details.dart

```diff
@@ -64,9 +63,7 @@ class FileDetails {
     }
   }
 
-  String get name => path.last;
```

Comment:
I think we should keep this getter.
It made a lot simpler would also keep the diff smaller